test/integration/regression/test_regression_bug390.py

```diff
@@ -31,7 +30,16 @@ def test():
         # fmt: on
     )
     r = pytester.runpytest_subprocess()
-    expected = ""
     actual = str(r.stderr)
 
-    assert actual == expected
+    # We can't assert here actual != "", because runpytest_subprocess prints warnings
+    # that can't be caught for Python 3.13 independent of the backend (pyodbc, turboodbc, websockets)
+    # since we moved from pytest-itde plugin to the pytest-backend plugin.
+    # The warnings look like the following:
+    #   <frozen importlib._bootstrap_external>:784: ResourceWarning: unclosed database in
+    #   <sqlite3.Connection object at 0x7faa81955b70>
+    #   ResourceWarning: Enable tracemalloc to get the object allocation traceback
+    #
+    # One possible reason could be that pytest-backend the integration-test-docker-environment
```

Comment:
This sentence is a bit hard to mentally parse. I think there is a verb,  which should be somewhere after `pytest-backend`, missing or maybe misplaced at the end of the sentence.